blindsign/blindrsa/brsa_test.go

```diff
@@ -20,8 +20,8 @@ import (
 )
 
 // 2048-bit RSA private key
-const testPrivateKey = `
------BEGIN RSA PRIVATE KEY-----
+var testPrivateKey = testingKey(`
```

Comment:
Addresses
```
::error file=blindsign/blindrsa/brsa_test.go,line=23,col=7::G101: Potential hardcoded credentials: RSA private key (gosec)
```

Follows a convention from the standard library (using a `testingKey` function like this).